public/app/features/dashboard/containers/DashboardPage.tsx

```diff
@@ -352,7 +352,8 @@ export class UnthemedDashboardPage extends PureComponent<Props, State> {
     const inspectPanel = this.getInspectPanel();
     const showSubMenu = !editPanel && !kioskMode && !this.props.queryParams.editview;
 
-    const toolbar = kioskMode !== KioskMode.Full && !queryParams.editview && (
+    const toolbar = ((kioskMode !== KioskMode.Full && !queryParams.editview) ||
+      (isPublic && !this.props.dashboard?.timepicker.hidden)) && (
```

Comment:
this does not feel right, why the timepicker check here?  it is already used in DashNav renderTimeControls

Reply:
Given the changes in https://github.com/grafana/grafana/pull/60689 for app chrome, think we need to step back and review other options to make this more maintainable, potentially a specfic page for public dashboard that is chromeless with a custom dashboard header to show dashboard title and time range (but no breadcrumbs links or mega menu)